src/modules/components/rv-jump-to/directives/rv-jump-to-directive.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+(function (angular) {
+
+  angular
+    .module('rv.components.RVJumpToModule')
+    .directive('rvJumpTo', rvJumpTo);
+
+  function rvJumpTo() {
+
+    return {
+      restrict: 'AE',
+      scope: {},
+      bindToController: {
+        fieldNames: '=',
+        totalSections: '@'
```

Comment:
Removed totalSections & modified the logic accordingly to remove it's dependency